Update to UI Components structure for Authenticator (#1386)
aws-amplify/docs: docs/ui/auth/sign-out.md

---
title: Sign Out
description: Sign Out component
filterKey: framework
---

> The simplest way to add authentication flows into your app is to use the Authenticator component.

Reviewer: ```suggestion
In order to add authentication flows into your application, use the Authenticator component.
```
